emoji/src/main/java/chat/rocket/android/emoji/EmojiKeyboardListener.kt

```diff
@@ -1,12 +1,12 @@
-package chat.rocket.android.widget.emoji
+package chat.rocket.android.emoji
 
 interface EmojiKeyboardListener {
     /**
      * When an emoji is selected on the picker.
      *
      * @param emoji The selected emoji
      */
-    fun onEmojiAdded(emoji: Emoji)
+    fun onEmojiAdded(emoji: Emoji) {}
```

Comment:
You don't need to use the brackets here.

Reply:
It's a Kotlin "feature". If you add a body to a function it kind of turns the interface into an adapter, so when a class implements it, this interface's function becomes optional to be overridden. In this specific case, we have places that we simply don't need `onEmojiAdded` to be implemented, which makes for less empty body overridden functions and thus, a cleaner code. :)

Reply:
@leonardoaramaki would be nice to add an _Optional_  in the method documentation.

Reply:
:+1: